modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -709,18 +709,22 @@ return view.extend({
 					so.depends('ra', 'hybrid');
 					so.depends('ra', 'relay');
 
-					so = ss.taboption('ipv6', form.ListValue, 'ra_management', _('DHCPv6-Mode'), _('Default is stateless + stateful<br />\
-						<ul style="list-style-type:none;">\
-						<li><strong>stateless</strong>: Router advertises prefixes, host uses <abbr title="Stateless Address Auto Config">SLAAC</abbr> \
-						to self assign its own address. No DHCPv6.</li>\
-						<li><strong>stateless + stateful</strong>: SLAAC. In addition, router assigns an IPv6 address to a host via DHCPv6.</li>\
-						<li><strong>stateful-only</strong>:  No SLAAC. Router assigns an IPv6 address to a host via DHCPv6.</li></ul>'));
-					so.value('0', _('stateless'));
-					so.value('1', _('stateless + stateful'));
-					so.value('2', _('stateful-only'));
+					so = ss.taboption('ipv6', form.Flag, 'ra_slaac', _('Enable <abbr title="Stateless Address Auto Config">SLAAC</abbr>'), _('Default is on.'));
```

Comment:
If the default is `on`, an `so.default = so.enabled` is missing. Or do you mean that it happens to be set to `1` in the defualt configuration? In this case you should remove the `Default is on` remark. The default state of a flag option should reflect the default state of the underlying software when no option is present in the configuration at all, regardless of what might be present in the default shipped configuration.

Reply:
OK now I have followed the default state of the software